dotnet/src/Agents/Core/ChatCompletionAgent.cs

```diff
@@ -25,6 +36,9 @@ public sealed class ChatCompletionAgent : ChatHistoryKernelAgent
         IReadOnlyList<ChatMessageContent> history,
         [EnumeratorCancellation] CancellationToken cancellationToken = default)
     {
+        // %%% TAO - CONSIDER THIS SECTION FOR LOGGING
+        this._logger.LogDebug("Agent {AgentId} invoked.", this.Id);
```

Comment:
This is channel-logic.  Let's pass in a logger please that is typed to the channel.  Please update `IChatHistoryHandler`

Also means the constructor and Ilogger go away